app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <dimen name="card_corner">20dp</dimen>
+    <dimen name="card_elevation">5dp</dimen>
+    <dimen name="image_width">70dp</dimen>
+    <dimen name="image_height">70dp</dimen>
+    <dimen name="image_padding_left">14dp</dimen>
+    <dimen name="image_padding_right">14dp</dimen>
+    <dimen name="temp_now_margin_bottom">2dp</dimen>
+    <dimen name="temp_now_text_size">24sp</dimen>
+    <dimen name="temp_min_max_margin_end">6dp</dimen>
+    <dimen name="temp_min_max_margin_right">6dp</dimen>
+    <dimen name="temp_min_max_text_size">20sp</dimen>
+    <dimen name="activity_main_text_view_height">70sp</dimen>
```

Comment:
activity_main_text_view_height, activity_main_image_view_width and activity_main_image_view_height should be in dp not sp

Reply:
Solved!